### PR TITLE
Implement basic documentation for ease of use

### PR DESCRIPTION
**Pull request by dddictionary**
This PR updates implements simple documentation on how to contribute to this project. 

The root `README.md` file gives steps to contribute, redirecting to another markdown file on getting the environment set up, and steps to run the bot itself. 

The `README.md` file in the `cogs` folder just has small details about what each cog is/does. 

The main concern now is the `README.md` in the `cogs` folder. I, to the best of my ability, tried to skim the files and write down what I understood they did. Mostly talked about the slash commands they implement as that seems the most straightforward way getting the point across. I left the cogs cogs that I didn't know about blank or not detailed. 

I still want to implement the todo idea, where a contributor can read the file to see which aspects of the bot need to be worked on, but still drawing blanks as to how to implement that. So far, I just have (WIP) next to the cog that is still in progress.

Please feel free to change any vocabulary mistakes, terminology, etc.

### CODE REVIEW

**Review by Arborym — APPROVED** [review]
Excellent work! Thank you for creating these 😃

File: cogs/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+## DeepL
+
+The DeepL cog implements the `translate` slash command.
+
+If you're confused as to why there are two different `translate` functions, one is to actually translate the text through the DeepL module while the other one is used to run service in a different thread. Note the use of `async`.
+
+## DetectAI (In Progress)
+
+The DetectAI cog implements the `detect-ai` slash command.
+
```
Comment (sebastiaan-daniels) [inline cogs/README.md:20]: Solution

File: cogs/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Uses a hacky soltuion to open a browser, submit the text to the form and get the result back. (Correct me if I'm wrong, but this is a WIP).
+
+## Dictionary
+
+The dictionary cog implements the `define` slash command, used to help define certain words. Uses more hacky HTML div element selction stuff to generate a response.
+
+Returns an markdown embed with the definition, and some links to the pronunciation.
+
```
Comment (sebastiaan-daniels) [inline cogs/README.md:28]: This class is indeed used to keep global track of how many messages are sent. This data is used by staffapplications

File: ENVIRONMENT.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Setting up the Environment
+
+1. Clone the forked repository onto your machine.
+2. Activate the virtual environment. On windows, running the [`setup.bat`](./setup.bat) file will suffice.
```
[the remark this answers is not in the document]
Comment (dddictionary) [reply ENVIRONMENT.md:4]: Sounds like a good idea to me. Will update the docs and remove the `.bat` file in the next commit.

File: cogs/README.md (hunk 1 above)
Comment (dddictionary) [reply cogs/README.md:20]: could you specify what you mean by solution?

Comment (dddictionary) [reply cogs/README.md:20]: oh, i see the mistake. thank you

**Review by sebastiaan-daniels — APPROVED** [review]

**Comment by Arborym** [comment]
Do not merge yet—I would like documentation surrounding the new `/detect-ai` command as well from #161
cc: @dddictionary

**Comment by dddictionary** [comment]
New push includes:
1. changes to the environment setup file.
   - Get rid of playwright command when downloading and setting up packages
   - New `.env` variables and their descriptions
2. changes to cogs folder's readme
   - added description for detect-ai cog. Explained how it works and the restrictions on role and channel.

**Review by sebastiaan-daniels — APPROVED** [review]

**Pull request merged**